help/robots.md

```diff
@@ -31,14 +33,13 @@ originally posted March 1994).
 
 Mindlessly downloading all of the URLs on this site will return
 terabytes of data. This has very real cost to us in terms of bandwidth
-consumed, and in terms of the responsiveness of our service for our many
-tens of thousands of real users.
+consumed, and in terms of the responsiveness of our service.
 
-This server is configured to monitor activity and deny access to sites
-that violate the above guidelines. Continued rapid-fire requests from
-any site after access has been denied (i.e. with *403 Access denied*
+arXiv monitors activity and will deny access to sites that violate these 
+guidelines. Continued rapid-fire requests from
+any site after access has been denied (i.e. with `403: Access denied`
 HTTP response) will be interpreted as an attack; and we will respond
-accordingly — without hesitation, and without further warning.
+accordingly — without hesitation or warning.
 

```

Comment:
I'm wondering if we should offer to provide additional guidance for folks attempting to use the arXiv corpus for research. Might be cool to have examples of code that have successfully bulk harvested in a controlled manner. Maybe there are enough examples out there in the community for harvesting arXiv?

Do we provide instructions for folks that get caught by robocop and are blocked via htaccess or firewall? I sometime worry we block legitimate researchers and they give up instead of trying to fix the issues with their tools and try again.

Reply:
I don't think anyone gives up, but also if they're blocked they can't access any pages but denied.html, so I don't think providing more guidelines here makes sense.